src/templates/docx/LDCETemplate.ts

```diff
@@ -46,7 +46,7 @@ export function LDCETemplate(passport: Passport): Document {
         fields: [
           {
             name: "Is there an agent?",
-            value: getBoolean("applicant.agent.form") ? "Yes" : "No",
+            value: get("applicant.agent.form"),
```

Comment:
This is just a "Yes" / "No" SetValue, no need for `getBoolean` here - see https://editor.planx.uk/opensystemslab/about-the-applicant